<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBuscaContato</class>
 <widget class="QDialog" name="DialogBuscaContato">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Contato</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblBuscarContato">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Digite o E-mail do contato a buscar:</string>
     </property>
     <property name="buddy">
      <cstring>txtBuscarContato</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtBuscarContato"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBuscarContato">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBuscarContato</sender>
   <signal>clicked()</signal>
   <receiver>DialogBuscaContato</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtBuscarContato</sender>
   <signal>returnPressed()</signal>
   <receiver>btnBuscarContato</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
